.github/workflows/gradle.yml

```diff
@@ -237,14 +237,15 @@ jobs:
       - name: Ensure no file change
         run: git --no-pager diff && test -z "$(git --no-pager diff)"
 
-      - name: Generate Test Report
-        uses: dorny/test-reporter@v1
```

Comment:
this was not working? I'm pretty sure I saw correctly generated test reports the other day. e.g. https://github.com/airbytehq/airbyte/runs/9003932342

Reply:
Ah, it doesn't work for some of the builds. It runs in three places: Platform, Connectors, E2E

For E2E it doesn't get truncated as you point out.

But for the platform build it gets trimmed entirely: https://github.com/airbytehq/airbyte/actions/runs/3290750324/jobs/5424309580

For Connectors it prints the summary and a few tests and then truncates the rest:
https://github.com/airbytehq/airbyte/actions/runs/3290750324/jobs/5424482385

If it makes sense we can restore the other two. Do we know if they are actively used? There's really no harm in turning them back on as they don't really take much time or resources

Reply:
After chatting with Davin on slack we decided to restore these.